src/store/store.ts

```diff
@@ -1,13 +1,23 @@
-import * as Y from 'yjs'
-import { TDBinding, TDShape } from '@tldraw/tldraw'
+import { Doc, Map, UndoManager } from 'yjs';
 import { WebsocketProvider } from 'y-websocket';
+import { TDBinding, TDShape } from '@tldraw/tldraw';
 
-export const doc = new Y.Doc()
+const defaultOptions = {
+	roomName: 'GLOBAL',
+	websocketUrl: 'ws://localhost:3345',
```

Comment:
I know it's not meant to be production-ready yet, but please remember to make it configurable in the future.